nipype/interfaces/ants/__init__.py

```diff
@@ -7,11 +7,13 @@
 from .registration import ANTS, Registration
 
 # Resampling Programs
-from .resampling import ApplyTransforms, ApplyTransformsToPoints, WarpImageMultiTransform, WarpTimeSeriesImageMultiTransform
+from .resampling import ApplyTransforms, ApplyTransformsToPoints, WarpImageMultiTransform, \
```

Comment:
instead of using a `\` we could use a tuple representation

```python
(ApplyTransforms, ApplyTransformsToPoints, WarpImageMultiTransform,
 WarpTimeSeriesImageMultiTransform)
```